src/demo.js

```diff
@@ -153,10 +153,24 @@ function dayInfo(day) {
   }
 }
 
+var DetailComponent = React.createClass({
+
+  render: function render(){
+    return (
+      <div>
+        <p>Sed posuere consectetur est at lobortis. Aenean eu leo quam.
```

Comment:
Could you put the event data in here? It is just the demo, but I think it would look more pro.